api/EXT_blend_minmax.json

```diff
@@ -5,40 +5,40 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/EXT_blend_minmax",
         "support": {
           "chrome": {
-            "version_added": true
+            "version_added": "38"
           },
           "chrome_android": {
-            "version_added": true
+            "version_added": "38"
           },
           "edge": {
             "version_added": "17"
           },
           "firefox": {
-            "version_added": "33"
+            "version_added": "47"
```

Comment:
Oops, no, 33 seems to have been correct: https://bugzilla.mozilla.org/show_bug.cgi?id=973815

Reply:
Actually, 33 is also incorrect.  The extension is not available in either macOS or Windows in Firefox 33.  However, there is a case of OS imparity: Windows was supported since 47, and macOS since 35.